index.html

```diff
@@ -48,5 +48,193 @@ <h1 class="title">I’m Mariane<span id="info" class="title">Glad to see you!</s
+                            <p class="subTitle">Canopy</p>
+                            <ul class="infoContainer">
+                                <li class="role">Back End Dev</li>
+                                <li class="year">2015</li>
+                            </ul>
+                        </div>
+                        <p class="projectDescription">A daily selection of privately personalized reads;
+                             no accounts or sign-ups required.                  
+                        </p>
+                        <div class="tags">
+                            <ul class="tagsFlex">
+                                <li class="html">HTML</li>
+                                <li class="css">CSS</li>
+                                <li class="javascript">Javascript</li>
+                            </ul>
+                        </div>
+                        <button type="button" class="viewProject">
+                            See Project
+                        </button>
+                    </div>
+                </li>
+            </ul>
+    
+        </div>
+    </section>
+
+    <section class="about-section" >
+            <h1 class="title">About<span>Myself</span>
+            </h1>
+    
+            <h6 class="paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.</h6>
+    
+            <h6 class="aboutline">LET’S CONNECT</h6>
+        
+                  <ul class="social-media">
+                    <li><a href=""><img class="icon twitter" 
+                        src="src/Twitter.png" alt="twitter-icon"/></a></li>
+    
+                    <li><a href=""><img class="icon linkedin" 
+                        src="src/Linkedin-icon.png" alt="linkedin-icon"/></a></li>
+    
+                    <li><a href=""><img class="icon msign" 
+                        src="src/m.png" alt="msign"/></a></li>
+    
+                    <li><a href=""><img class="icon github" 
+                        src="src/Github.png" alt="github-icon"/></a></li>
+    
+                    <li><a href=""><img class="icon finger" 
+                        src="src/Angel.png" alt="AngelList"/></a></li>
+                  </ul>
+    
+                <button type="button" class="resumeButton">Get my resume</button> 
+                
+                <div class="mySkills">
+                    <ul class="skills">
+                      <li>
+                        <section class="skillsDropdown"><!-- Future Dropdown Menu -->
+                          <h3 class="skillsTitle">Languages</h3>
+                          <img src="src/arrow-down.png" alt="arrowdown" />
+                        </section>
+                        <ul class="skillsDesc">
+                          <li class="skillsDescList">
```

Comment:
- [ ] You did a good job adding the skills section but please make sure to follow [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) for this section too. Please fix it. Here you have a screenshot of the skills section:

![image](https://user-images.githubusercontent.com/74506933/153707982-f17410ca-d534-4ff5-92bf-7aff859aa39f.png)